### DISCUSSION TITLE
새로운 기술 적용 검토

### DISCUSSION THREAD

**Discussion by locodingve** [prgrms-web-devcourse/slam-server#134] [General]
새로운 기술 적용과 관련된 내용을 논의하는 장입니다.

**Comment by locodingve** [comment]
젤리~ 
우선 말씀 드리기 전에 "적용해 주세요!" 라고 말하는 것이 **아니라는 것**을 강조드립니다. 

로그인 서비스에도 레디스를 활용하여 데이터를 캐싱하더라구요~
그래서 혹시나 작업하시는데 아이디어가 될 수 있을 것 같아서, 공유드립니다. 

나중에 논의 드릴 예정인데, 레디스를 활용하는 것에 대해서 백앤드 개발자 분들과 논의 해보고 싶어요~
좋은 경험이 되실 것 같아서요. 아마 헤이(court domain) 부분에 많이 활용될 것 같아요~ (헤이 쪽은 ㅋㅋㅋ 강한 의견드립니당 히히 )
사용하는 것으로 결정나면, 사용 예정 기간에 맞춰서 세팅해 놓겠습니다~

**Reply by KwonYeKyeong** [reply]
이번주까지 레디스에 대한 사전 조사 및 공부를 하겠습니다!! 
공부 후에 적용에 대해서 말씀드려도 될까요?? 좋은 아이디어💡 감사합니다~~

**Reply by locodingve** [reply]
넵~ 저도 이번주는 어려울 것 같고, 서버 이전과 notification 테스트 코드 완료되면 공부할게요~

우선 지금 springboot redis 연동해서 cache 처럼 사용하게끔은 한 번 짜보고 테스트해보았어요~ 
구체적으로 공부는 다음주 수욜 이후 부터 가능할 것 같습니다. 
화이팅~!!👍
